docs/src/app_dev/ledger_api/index.rst

```diff
@@ -8,5 +8,20 @@
 Ledger API
 ==========
 
-Current documentation for the Ledger API can be found `here <https://docs.digitalasset.com/build/3.3/reference/app-dev/ledger-api/>`_.
+As can be seen from the :ref:`validator-network-diagram` shown below, the Ledger API is
+served by the Canton Participant node running as part of in a Validator Node.
+
+..
+   _LucidChart link: https://lucid.app/lucidchart/46af6ed2-25b8-4608-88f7-66fc19674fce/edit?viewport_loc=-5217%2C-3874%2C6157%2C3382%2C0_0&invitationId=inv_fc3a3c02-e9fd-4ca9-9388-8defdfaa7153
+
+
+.. image:: ../../validator_operator/images/validator-network-diagram.png
+  :width: 600
+  :alt: Validator Network Diagram
```

Comment:
I don't think the "Validator Network Diagram" is sufficient.  The VND is an infrastructure diagram. It doesn't describe the software architecture and the API endpoints.    I realized this when I asked "Where is the application?" in the picture that is about "how to deploy a validator using Helm charts."  I have a very high conviction that this is needed.

Reply:
Good point. Built and added a better diagram.